dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/mvc/messageconverter/XmlPathMappingJackson2XmlHttpMessageConverter.java

```diff
@@ -52,6 +52,7 @@
         Pattern.compile( "/(\\d\\d/)?events(/?$|/.+)" ),
         Pattern.compile( "/(\\d\\d/)?trackedEntityInstances(/?$|/.+)" ),
         Pattern.compile( "/(\\d\\d/)?dataValueSets(/?$|/.+)" ),
+        Pattern.compile( "/analytics/dataValueSet.xml" ),
```

Comment:
Will this handle versioned endpoints without the (\\d\\d/)? pattern.
In general, shouldn't it align with the others?
e.g.
"/(\\d\\d/)?dataValueSet(/?$|/.+)"
?

Reply:
Sorry - I see Lars already made this comment.

Reply:
Done. Versions included.